README.md

```diff
@@ -49,6 +49,13 @@ apt 'com.ryanharter.auto.value:auto-value-gson:0.2.2'
 
 (Using the [android-apt](https://bitbucket.org/hvisser/android-apt) plugin)
 
+In Order for the latest update to work (where this documentation is referring to) you need to have the snapshot
```

Comment:
Make sense, but I the documentation should refer to a valid gradle dependency, which works with the provided example.